### PR TITLE
Remove v1 of all IAM users

### PR DESCRIPTION
**Pull request by rogeruiz**
All these users have been rotated to `v2` / `_curr` values across all pipelines.

I will merge this and bootstrap the environments.

### CODE REVIEW

**Comment by rogeruiz** [comment]
Gonna start before merging by deactivating all the previous IAM crews manually and monitor the CI. I've double-triple checked all the crews currently flown, but I want to make absolutely sure 🤞

**Comment by jmcarp** [comment]
Looks like this doesn't actually delete credentials--it just hides them from terraform outputs. This should also finish the cdn broker credentials rotation.

**Comment by rogeruiz** [comment]
Alright let me verify all the concourse creds are right and double check EW.

**Comment by rogeruiz** [comment]
Thanks @jmcarp I'll remove the versions as well. Marked this WIP until for now.

- [ ] verify EW
- [ ] actually delete `_v2` keys in Terraform files.

**Comment by wjwoodson** [comment]
Deleted all _v1 keys from Terraform files @rogeruiz and I are still in the process of verifying they're cleared from pipelines

**Comment by rogeruiz** [comment]
Verified that the users are no longer in use in fly or manifests

File: terraform/stacks/external/outputs.tf [hunk 1]
```diff
@@ -3,10 +3,10 @@ output "cdn_broker_username" {
   value = "${module.cdn_broker.username}"
 }
 output "cdn_broker_access_key_id_prev" {
-  value = "${module.cdn_broker.access_key_id_prev}"
+  value = ""
```
Comment (jmcarp) [inline terraform/stacks/external/outputs.tf]: Why change this? Looks like the output from the module is already blank.

Comment (rogeruiz) [inline terraform/stacks/external/outputs.tf]: ah okay. so we don't need to do it both places. let me revert this then.

Comment (rogeruiz) [inline terraform/stacks/external/outputs.tf]: so does the same thing revert need to happen for `terraform/modules/cloudfoundry/outputs.tf` and `terraform/stacks/main/outputs.tf` as well? @jmcarp

Comment (jmcarp) [inline terraform/stacks/external/outputs.tf]: That's the idea--we should only have to blank out each credential pair in one place.

**Pull request merged**